backend/app/api/ws/chat_namespace.py

```
@@ -190,6 +190,43 @@ async def _finalize_failed_ai_trigger(
+    team_ref = (task_json.get("spec") or {}).get("teamRef")
+    if not team_ref:
+        return client_team
+
+    bound_team = (
+        db.query(Kind)
+        .filter(
+            Kind.kind == "Team",
+            Kind.name == team_ref.get("name"),
+            Kind.namespace == team_ref.get("namespace", "default"),
+            Kind.user_id == team_ref.get("user_id", existing_task.user_id),
+            Kind.is_active == True,
```

Ruff (0.16.3)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)